base/applications/regedit/lang/ro-RO.rc

```diff
@@ -83,57 +83,57 @@ BEGIN
         MENUITEM "Imp&ortare…", ID_REGISTRY_IMPORTREGISTRYFILE
         MENUITEM "E&xportare…", ID_REGISTRY_EXPORTREGISTRYFILE
         MENUITEM SEPARATOR
-        MENUITEM "Î&ncărcare registru…", ID_REGISTRY_LOADHIVE, GRAYED
-        MENUITEM "Eli&berare registru…", ID_REGISTRY_UNLOADHIVE, GRAYED
+        MENUITEM "Î&ncărcare de regiștri…", ID_REGISTRY_LOADHIVE, GRAYED
+        MENUITEM "&Descărcare de regiștri…", ID_REGISTRY_UNLOADHIVE, GRAYED
         MENUITEM SEPARATOR
         MENUITEM "&Conectare la un registru din rețea…", ID_REGISTRY_CONNECTNETWORKREGISTRY
-        MENUITEM "&Deconectare de la registru din rețea…", ID_REGISTRY_DISCONNECTNETWORKREGISTRY, GRAYED
+        MENUITEM "&Deconectare de la regiștrii din rețea…", ID_REGISTRY_DISCONNECTNETWORKREGISTRY, GRAYED
```

Comment:
"&Deconectare de la regiștrii din rețea…" collides with MENUITEM "&Descărcare de regiștri…". Change is mandatory.

Reply:
resolved.